.github/workflows/main.yml

```diff
@@ -43,3 +43,20 @@ jobs:
         continue-on-error: false
         env:
           LIGHTGBM_EXAMPLES_PATH: LightGBM-2.3.1
+ docs:
```

Comment:
Do we want to restrict this to running on master?

Reply:
I think only execution on the master is enough.